### PR TITLE
add record device to dev

### PR DESCRIPTION
**Pull request by justbill2020**
####  Description

Eliminates the need to configure .asoundrc file to get microphone working... this will use arecord and manually assign the capture device in the arecord cmd line. the recording device is configured in the remote configUI under speech settings

####  Fixes Related issues
- add related
- issues here

####  Checklist
- [x] I have read and understand the CONTRIBUTIONS.md file
- [x] I have searched for and linked related issues
- [x] I have added config.schema.json file if config option are required.
- [x] I am NOT targeting master branch

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
     try{
       config = require('./config.json')
     } catch (e){
-      config = require('./config.default.json')
+      config = require('./remote/.config.default.json')
```
Comment (evancohen) [inline index.html]: Ooh, good idea making this a dotfile

File: remote/.config.default.json [hunk 2]
```diff
@@ -79,23 +79,5 @@
...
-        "authorizationPath": "/oauth2/authorize",
-        "tokenUri": "https://api.fitbit.com",
-        "tokenPath": "/oauth2/token"
-      },
-      "authorization_uri": {
-        "redirect_uri": "http://localhost:4000/fitbit_auth_callback/",
-        "response_type": "code",
-        "scope": "activity nutrition profile settings sleep social weight heartrate",
-        "state": "3(#0/!~"
-      }
```
Comment (evancohen) [inline remote/.config.default.json]: Did you mean to remove the fitbit and todoist stuff from here?

File: sonus.js [hunk 3]
```diff
@@ -38,7 +37,8 @@ for (let i = 0; i < config.speech.hotwords.length; i++) {
 
 const language = config.general.language
 const recordProgram = (os.arch() == 'arm') ? "arecord" : "rec"
-const sonus = Sonus.init({ hotwords, language, recordProgram }, speech)
+const device = (config.speech.device != "") ? config.speech.device : 'default'
+const sonus = Sonus.init({ hotwords, language, recordProgram, device }, speech)
```
Comment (evancohen) [inline sonus.js]: Radical

File: remote/.config.default.json (hunk 2 above)
Comment (justbill2020) [inline remote/.config.default.json]: nope

File: index.html (hunk 1 above)
Comment (justbill2020) [inline index.html]: i thought it was... lol

File: sonus.js (hunk 3 above)
Comment (justbill2020) [inline sonus.js]: that's the part you changed... lol the rest is the cool shit... dynamic drop down for recording device in config UI

**Pull request merged**